Src/StdLib/Lib/_weakrefset.py

```diff
@@ -70,9 +70,9 @@ def __len__(self):
     def __contains__(self, item):
         try:
             wr = ref(item)
+            return wr in self.data
```

Comment:
Changes to the stdlib need to be reviewed quite a bit, we want to stay as close to the CPython stdlib as possible. In some cases, it's not possible, but we want to make sure that is the case before incorporating changes.

Reply:
I completely understand. I’m surprised that line isn’t an issue in CPython - perhaps cpython couldnt context switch between the two lines? I don’t really understand cpython’s thread model enough to comment. From .nets perspective there’s definitely a potential race condition there tho

I guess the new structure is functionally the same as the old in this case so I could potentially submit a CPython pull request also (probably get rejected but you never know)